tools/flexynesis/flexynesis.xml

```diff
@@ -0,0 +1,596 @@
+        ln -s '$test_clin' input/test/clin.csv &&
+
+        #if str($assay_main) != '':
+            #set $name = str($assay_main)
+            ln -s '$train_omics_main' input/train/${name}.csv &&
+            ln -s '$test_omics_main' input/test/${name}.csv &&
+            #set $data_names = [str($assay_main)]      
+        #else
+            ln -s '$train_omics_main' input/train/main.csv &&
+            ln -s '$test_omics_main' input/test/main.csv &&
+            #set $data_names = ['main']
+        #end if
+        #for $i, $element in enumerate($omics)
+            ## #if ${element.train_omics} and ${element.test_omics}:
+                #if str($element.assay) != '':
+                    #set $i = str($element.assay)
+                #end if
+                ln -s '${element.train_omics}' input/train/omics_${i}.csv &&
+                ln -s '${element.test_omics}' input/test/omics_${i}.csv &&
+                $data_names.append("omics_" + str($i))
+            ## #end if
+        #end for
+        flexynesis  
+                    --data_path 'input'
+                    --outdir 'output'
+                    --model_class $model_class
+                    #if str($model_class) == 'GNN':
+                        --gnn_conv_type $gnn_conv_type
+                        --string_organism $string_organism
+                        --string_node_name $string_node_name
+                    #end if
+                    #if str($training_type.model) == 's_train':
+                        #if str($target_variables) != '':
+                            --target_variables $target_variables
+                        #end if
+                        #if str($surv_event_var) != '':
+                            --surv_event_var $surv_event_var
+                            --surv_time_var $surv_time_var
+                        #end if
+                    #end if
+                    #if str($training_type.model) == 'cm_train':
+                        --input_layers $input_layers
+                        --output_layers $output_layers
+                    #end if
+                    --fusion_type $fusion_type
+                    --hpo_iter $hpo_iter
+                    --finetuning_samples $finetuning_samples
+                    --variance_threshold $variance_threshold
+                    --correlation_threshold $correlation_threshold
+                    --subsample $subsample
+                    --features_min $features_min
+                    --features_top_percentile $features_top_percentile
+                    --data_types $str(",".join($data_names))
+                    --early_stop_patience $early_stop_patience
+                    --hpo_patience $hpo_patience
+                    $log_transform
+                    $use_loss_weighting
+                    $use_cv
+                    $evaluate_baseline_performance
+                    $disable_marker_finding
+                    ## "\${GALAXY_FLEXYNESIS_EXTRA_ARGUMENTS:-''}"
```

Comment:
```suggestion
                    "\${GALAXY_FLEXYNESIS_EXTRA_ARGUMENTS:-''}"
```

Reply:
I am still working on that. It creates empty space and give the error: `invalid argument: `